server/sonar-webserver-webapi/src/main/java/org/sonar/server/qualityprofile/ws/DeactivateRuleAction.java

```diff
@@ -88,8 +88,8 @@ public void handle(Request request, Response response) throws Exception {
       QProfileDto profile = wsSupport.getProfile(dbSession, QProfileReference.fromKey(qualityProfileKey));
       OrganizationDto organization = wsSupport.getOrganization(dbSession, profile);
       wsSupport.checkCanEdit(dbSession, organization, profile);
-      logger.info("Deactivate rule from QProfile Request :: ruleUuid : {}, qProfileUuid :{} and organization : {}, user : {}",
-              rule.getUuid(), profile.getRulesProfileUuid(), organization.getUuid(), userSession.getLogin());
+      logger.info("Deactivate rule from QProfile:: organization: {}, qProfile: {}, language: {}, rule: {}, user: {}",
+              organization.getKey(), profile.getName(), profile.getLanguage(), rule.getUuid(), userSession.getLogin());
```

Comment:
@arunsimha-cs  We are logging the rule uuid. This mean user should have access to both logs and the database to determine the deactivated rule. Is this log for audit purpose or will it be used in any alerts? Any info on this?